packages/experimental/Badge/src/BadgeTokens.ts

```diff
@@ -3,6 +3,11 @@ import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { globalTokens } from '@fluentui-react-native/theme-tokens';
 import { BadgeTokens } from './Badge.types';
 
+const badgeShadowStyle = {
+  ambient: { x: 0, y: 8, blur: 8, color: '#00000024' },
+  key: { x: 0, y: 0, blur: 1, color: '#0000001f' },
+};
```

Comment:
Does the right token not exist yet

Reply:
I got this from notification component. Currently Shadow doesn't work for Badge, that's why I created draft PR so Lynn can take a look on it. Badge is not visible when Shadow is shown:
![image](https://user-images.githubusercontent.com/11574680/187290136-abbdb15d-e9c1-4f02-b14a-22d943e67510.png)

Reply:
I got the token from the theme but it's still not visible

Reply:
I think that will be a win32 issue that is outside Lynn's original scope, so you may have to route the bug to someone else in cxe. We only validated shadow for iOS and macOS for the initial component

Reply:
Talked to Lynn offline and got a better idea of the bug. Seems it might not be as simple as a win32 netui problem. My apologies..